.circleci/config.yml

```diff
@@ -892,11 +892,10 @@ workflows:
           name: "unit-s_base-lin-py39"
           image: "python:3.9"
           toxenv: "py39"
-# Enable when circle offers python 3.10 image
-#      - test:
-#         name: "unit-s_base-lin-py310"
-#         image: "python:3.10"
-#         toxenv: "py310"
+      - test:
```

Comment:
should we add matrix logic for these unit-s_base jobs as well?

Reply:
probably not as the "matrix" will be too sparse :) let's revisit before the next release - I'm cooking a PR that will reorganize testing a bit to speed things up by converting/splitting our parallel jobs into a proper dag with certain jobs depending on each other.